frida_mode/Makefile

```diff
@@ -0,0 +1,137 @@
+PWD:=$(shell pwd)/
+INC_DIR:=$(PWD)inc/
+SRC_DIR:=$(PWD)src/
+INCLUDES:=$(wildcard $(INC_DIR)*.h)
+SOURCES:=$(wildcard $(SRC_DIR)*.c)
+BUILD_DIR:=$(PWD)build/
+
+FRIDA_BUILD_DIR:=$(BUILD_DIR)frida/
+FRIDA_TRACE:=$(FRIDA_BUILD_DIR)afl-frida-trace
+
+GUM_DEVKIT_ARCH=x86_64
```

Comment:
The devkit version should depend on the current arch, right?

Reply:
Yes

Reply:
Do you suggest I perhaps use the variable ARCH, document it as needing to be set in README.md and perhaps default to x86_64 if not set? Happy to go with whatever you think best.

Reply:
maybe just `$(shell uname -m)` ?

Reply:
```
dave@mini AFLplusplus % echo $(shell uname -m)
zsh: command not found: shell

dave@mini AFLplusplus % echo $(uname -m) 
arm64
```

Reply:
Fixed